README.Mac.md

```diff
@@ -1,294 +1,281 @@
-# LORIS Mac Install Guide
-### Mac is no longer supported as of 15.10. 
+### These instructions last tested for LORIS version v20.*
+# Install a developer instance of LORIS on macOS
+### We recommend installing LORIS on Ubuntu or CentOS
+#### macOS is no longer supported for "production" as of v15.10. 
 
-This is a guide on how to get a LORIS instance Running on your Mac computer. It is designed for installation in a sandbox environment. 
-It has been partially tested for Mac OS X 10.9 and 10.10 for 15.04. Installing on Mac OS X El Capitan will require additional Apache setup. 
-For best results, we recommend installing LORIS on Ubuntu or CentOS.
+This is a guide on how to get a LORIS instance running on your Apple computer for development. You're expected to have basic understanding of using a terminal and editing configuration files. 
+For production, we recommend installing LORIS on Ubuntu or CentOS.
 
 # 1. System Requirements
 
-Before continuing to the subsequent sections ensure that the following tools are installed.  Sudo permission is required.  This section covers installation and recommended versions of:
+The following list provides foresight into the proceeding sections and recommended versions to install:
+
+ * macOS (tested on Mojave 10.14.3)
+ * XCode Command Line Tools
```

Comment:
This and Xdebug are optional right? Can you separate these programs from the required software, maybe in their own `Optional` header beneath the last bullet? It would also be good to highlight that these are optional in the headers below e.g. step 1.7.

Reply:
@johnsaigle Xcode Command Line Tools are required to build some of the packages (not really optional). I think most developers will desire using Xdebug for debugging. It might be good to leave the Xdebug snippet where it is in the guide because the flow of installing makes sense to install it after doing the php addition to apache.

Reply:
That's fine but maybe just tag step 1.7 as optional.

Reply:
@johnsaigle I added the optional tag.